source/documentation/other-topics/custom-domain-cert.md

```diff
@@ -48,16 +55,16 @@ material will be stored, in your namespace.
        - domains:
          - '<hostname>'
          dns01:
-           provider: <provider>
+           provider: route53-cloud-platform
    ```
 
-3. Make sure the certificate has been issued correctly, by checking its `Status`:
+2. Make sure the certificate has been issued correctly, by checking its `Status`:
```

Comment:
What does a good status look like here? Are there any common errors the user should look out for?